08-components-lab/src/01-timeconverter.jsx

```diff
@@ -1,12 +1,60 @@
-/**
- * Lab 01: 
- *
- * Implement a time converter component.
- * Component should show 3 input fields labeled hours, minutes and seconds.
- * Typing a value in either field updates the other two with corresponding values.
- */
-
-import React from 'react';
-
-export default React.createClass({
-});
+/**
+ * Lab 01:
+ *
+ * Implement a time converter component.
+ * Component should show 3 input fields labeled hours, minutes and seconds.
+ * Typing a value in either field updates the other two with corresponding values.
+ */
+import React from "react";
+
+export default class TimeConverter extends React.Component {
+	constructor() {
+		super();
+
+		this.state = {
+			hours: 0, minutes: 0, seconds: 0,
+		};
+
+		// bind functions for the changes in each input field
+		this.hoursChanged = this.hoursChanged.bind(this);
+		this.minutesChanged = this.minutesChanged.bind(this);
+		this.secondsChanged = this.secondsChanged.bind(this);
+	}
+
+	hoursChanged(event) {

```

Comment:
you actually need only one method - seconds changed. And the rest are derived from it